ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -28,16 +49,201 @@ import {
   CardTitle,
 } from '@/components/ui/card';
 import { Label } from '@/components/ui/label';
+import { calcOverallVulnerability } from '@/helpers/calc-overall-vulnerability';
+import { getVulnerabilityRatingBackgroundColor } from '@/helpers/get-status-colors';
+import { paginationSchema } from '@/schemas';
+
+const defaultPageSize = 10;
+
+const columns: ColumnDef<VulnerabilityWithIdentifier>[] = [
+  {
+    accessorKey: 'package',
+    header: 'Package',
+    cell: ({ row }) => {
+      const type = row.original.identifier.type;
+      const namespace = row.original.identifier.namespace;
+      const name = row.original.identifier.name;
+      const version = row.original.identifier.version;
+
+      return (
+        <div className='font-semibold'>
+          {type ? type.concat(':') : ''}
+          {namespace ? namespace.concat('/') : ''}
+          {name ? name : ''}
+          {version ? '@'.concat(version) : ''}
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: 'externalId',
+    header: 'External ID',
+    cell: ({ row }) => (
+      <Badge className='bg-gray-400'>
+        {row.original.vulnerability.externalId}
+      </Badge>
```

Comment:
Would it be too wide if this didn't break the line? IMO it would look better as it's a badge with one string.